script.js

```diff
@@ -0,0 +1,37 @@
+var box = document.querySelectorAll(".box");
```

Comment:
You could also do `document.querySelectorAll("td")`. 

It couldn't hurt to *try* jQuery!